desktop/src/gui/dialogs.rs

```diff
@@ -42,32 +44,37 @@ impl Dialogs {
         window: Weak<winit::window::Window>,
         event_loop: EventLoopProxy<RuffleEvent>,
     ) -> Self {
+        let picker = FilePicker::new(window, preferences.language());
```

Comment:
This does not seem to allow for language changes at runtime (i.e. changing it in preferences)

Reply:
Should be good now!